neuralhydrology/datasetzoo/__init__.py

```diff
@@ -20,19 +23,21 @@ def get_dataset(cfg: Config,
                 scaler: dict = {}) -> BaseDataset:
     """Get data set instance, depending on the run configuration.
 
-    Currently implemented datasets are 'caravan', 'camels_aus', 'camels_br', 'camels_cl', 'camels_gb', 'camels_us', and 
+    Currently implemented datasets by default are 'caravan', 'camels_aus', 'camels_br', 'camels_cl', 'camels_gb', 'camels_us', and
```

Comment:
I think the "by default" can be removed. It is not a "default", it is a fact, no? Your other sentence clarifies that additional dataset classes can be added with the register_dataset function.